cloudsql-mysql-plugin/src/e2e-test/java/io/cdap/plugin/CloudMySqlClient.java

```diff
@@ -0,0 +1,183 @@
+package io.cdap.plugin;
+
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import org.junit.Assert;
+
+import java.sql.*;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.TimeZone;
+
+public class CloudMySqlClient {
+
+    private static final String database = PluginPropertyUtils.pluginProp("DatabaseName");
+    private static final String connectionName = PluginPropertyUtils.pluginProp("ConnectionName");
+
+    public static Connection getCloudSqlConnection() throws ClassNotFoundException, SQLException {
+        Class.forName("com.google.cloud.sql.mysql.SocketFactory");
+        String instanceConnectionName = System.getenv("CLOUDSQLMYSQL_CONNECTIONNAME");
+        String database = PluginPropertyUtils.pluginProp("DatabaseName");
+        String username = System.getenv("CLOUDSQLMYSQL_USERNAME");
+        String password = System.getenv("CLOUDSQLMYSQL_PASSWORD");
+
+        String jdbcUrl = String.format(
+                PluginPropertyUtils.pluginProp("jdbcURL"),
+                database, instanceConnectionName, username, password);
```

Comment:
fixing spacing in above lines.

Reply:
done
commit link- b03766131af7a687ad66f495a6eac551f74c15d8